packages/transaction-controller/src/gas-flows/LineaGasFeeFlow.ts

```diff
@@ -129,64 +115,27 @@ export class LineaGasFeeFlow implements GasFeeFlow {
         to: transactionMeta.txParams.to,
         value: transactionMeta.txParams.value,
         input: transactionMeta.txParams.data,
+        // Required in request but no impact on response.
         gasPrice: '0x100000000',
       },
     ]);
   }
 
-  #getBaseFees(lineaResponse: LineaEstimateGasResponse): FeesByLevel {
-    const baseFeeLow = hexToBN(lineaResponse.baseFeePerGas);
-    const baseFeeMedium = baseFeeLow.muln(BASE_FEE_MULTIPLIERS.medium);
-    const baseFeeHigh = baseFeeLow.muln(BASE_FEE_MULTIPLIERS.high);
-
-    return {
-      low: baseFeeLow,
-      medium: baseFeeMedium,
-      high: baseFeeHigh,
-    };
-  }
-
-  #getPriorityFees(
-    lineaResponse: LineaEstimateGasResponse,
-    gasFeeEstimates: GasFeeControllerEstimates,
+  #getValuesFromMultipliers(
+    value: Hex,
+    multipliers: { low: number; medium: number; high: number },
   ): FeesByLevel {
-    const mediumPriorityIncrease = this.#getPriorityLevelDifference(
-      gasFeeEstimates,
-      GasFeeEstimateLevel.medium,
-      GasFeeEstimateLevel.low,
-    );
-
-    const highPriorityIncrease = this.#getPriorityLevelDifference(
-      gasFeeEstimates,
-      GasFeeEstimateLevel.high,
-      GasFeeEstimateLevel.medium,
-    );
-
-    const priorityFeeLow = hexToBN(lineaResponse.priorityFeePerGas);
-    const priorityFeeMedium = priorityFeeLow.add(mediumPriorityIncrease);
-    const priorityFeeHigh = priorityFeeMedium.add(highPriorityIncrease);
+    const low = hexToBN(value).muln(multipliers.low);
+    const medium = low.muln(multipliers.medium);
+    const high = low.muln(multipliers.high);
```

Comment:
If `multipliers.low != 1` then I think `medium` and `high` will have an extra multiplier. 

```suggestion
    const base = hexToBN(value);
    const low = base.muln(multipliers.low);
    const medium = base.muln(multipliers.medium);
    const high = base.muln(multipliers.high);
```

Reply:
Great spot! Done.